packages/checkbox/test/checkbox.test.ts

```diff
@@ -120,7 +121,10 @@ describe('Checkbox', () => {
         ) as Checkbox;
 
         expect(autoElement).to.exist;
-        expect(document.activeElement).to.equal(autoElement);
+        await waitUntil(
+            () => document.activeElement === autoElement,
+            'Autofocused'
+        );
```

Comment:
Support the move to lazy focusability by waiting for the focus to be applied rather than expecting it synchronously.